health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/AdvancedMapCharts/LatLongMap/LatLongMapChart.js

```
@@ -0,0 +1,288 @@
+    });
+    if (isLoading) {
+      return <Loader />;
+    }
+    response?.responseData?.data?.forEach((d) => {
+      let plotLabel = response?.responseData?.plotLabel;
+      d?.plots?.forEach((p) => {
+        let obj = {};
+        let plotName = p.name;
+        obj[plotLabel] = p.name;
+        obj[d.headerName] = p.value;
+        if (tableData.current.hasOwnProperty(plotName)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)